src/components/tasks/card/index.tsx

```diff
@@ -12,19 +12,17 @@ import {
 const moment = require('moment');
 
 type Props = {
-  content: task,
+  content: task;
 };
 
 const Card: FC<Props> = ({ content }) => {
   const {
-    title,
-    endsOn,
-    startedOn,
-    status,
-    assignee,
+    title, endsOn, startedOn, status, assignee,
   } = content;
 
-  const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${assignee}/img.png`);
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(
+    `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${assignee}/img.png`,
```

Comment:
While you are at it, can you remove the last `/` from the `NEXT_PUBLIC_GITHUB_IMAGE_URL` variable?

Files -
https://github.com/Real-Dev-Squad/website-status/blob/develop/.env.development
https://github.com/Real-Dev-Squad/website-status/blob/develop/.env.production

Reply:
Also please revert the indentation change if not necessary

Reply:
indentation done. /'s removed. Please check.